packages/components/src/modal/index.js

```diff
@@ -162,6 +157,12 @@ function Modal( props, forwardedRef ) {
 							'has-scrolled-content': hasScrolledContent,
 						} ) }
 						role="document"
+						tabIndex="0"
```

Comment:
I'd explore what `hasScrolledContent` is and maybe set the tabindex attribute conditionally based on that. It would be nice to avoid a tab stop when there's no need for it. Also, e should consider that this is a silent tab stop, as nothing will be announced. What would be the best way to label it without making it a landmark?

Reply:
It should not be a tab stop since I moved the focus hooks.